helm_deploy/cla-public/values-staging.yaml

```diff
@@ -6,7 +6,7 @@ image:
   pullPolicy: IfNotPresent
 
 environment: staging
-backend_base_uri: https://staging-backend.cla.dsd.io/
```

Comment:
I think this should not have had the the trailing slash as the paths are passed with the leading slash which results in the the following schema://domain.tld//uri which django will see it as a non-existing path and return a 404.

This wasn't happening on the current deployment system because nginx was sitting in front of uswgi and reconstructing the url.